api/api.proto

```diff
@@ -2207,24 +2215,32 @@ message SdkCloudBackupStatus {
   google.protobuf.Timestamp completed_time = 6;
   // NodeID is the ID of the node where this Op is active
   string node_id = 7;
+  // SourceVolumeID is the the volume that is either being backed up to cloud
+  // or target volume to which a backup is being restored
+  string src_volume_id = 8;
+  // Info currently indicates the failure cause for failed backup/restore
+  repeated string info = 9;
 }
 
 // Defines a request to retreive the status of a backup or restore for a
 // specified volume
 message SdkCloudBackupStatusRequest {
-  // This is an optional value which is used to get information on the
-  // status of a backup for the specified volume. If no volume id is provided,
-  // then status for all volumes is returned.
+  // VolumeId is an optional value which is used to get information on the
+  // status of a backup for the specified volume. If no volume id and Name
```

Comment:
Is this supposed to be `If no volume id or name ...` ?

Reply:
yeah, name.